dev/AutoSuggestBox/AutoSuggestBox.vcxitems

```diff
@@ -15,12 +15,12 @@
   </ItemGroup>
   <ItemGroup>
     <Page Include="$(MSBuildThisFileDirectory)AutoSuggestBox_themeresources.xaml">
-      <UseVisualStyle>Latest</UseVisualStyle>
+      <ControlsResourcesVersion>Version2</ControlsResourcesVersion>
```

Comment:
@CanHua can you open an issue to fix up the createnewcontrols script to reflect these changes ?

Reply:
where is the script? I didn't see any other UseVisualStyle in the repro

Reply:
Its the *NewControlInProject* and *NewControlProject* scripts here - https://github.com/microsoft/microsoft-ui-xaml/tree/master/tools.

Reply:
#4291